src/app/song/song-list/song-list.component.ts

```diff
@@ -33,6 +38,11 @@ export class SongListComponent implements OnInit {
     this.selectedSong = song;
   }
 
+  onChangePage(pageData: PageEvent){
+    this.currentPage = pageData.pageIndex + 1;
```

Comment:
Why +1. If this funx only for moving forward?
How do we go back

Reply:
its not for moving forward. it is because the page index starts from 0 but the backend expect it to begin from 1.